src/utils/achievements.ts

```diff
@@ -47,24 +46,34 @@ export const getAchievementDescription = (campaign: Campaign): TranslatableText
  */
 export const getClaimableIfoData = async (account: string): Promise<Achievement[]> => {
   const ifoCampaigns = ifosList.filter((ifoItem) => ifoItem.campaignId !== undefined)
-  const ifoCampaignAddresses = ifoCampaigns.map((ifoItem) => ifoItem.address)
-  const pointCenterContract = getPointCenterIfoContract()
 
   // Returns the claim status of every IFO with a campaign ID
-  let claimStatuses = []
-  try {
-    claimStatuses = (await pointCenterContract.checkClaimStatuses(account, ifoCampaignAddresses)) as boolean[]
-  } catch (error) {
-    console.error(error)
-  }
+  const claimStatusCalls = ifoCampaigns.map(({ address }) => {
```

Comment:
you can use a shortcut without the return

Reply:
Yeah...I have been avoiding doing that. I like the explicit return recently.